package.json

```diff
@@ -5,8 +5,8 @@
   "private": true,
   "scripts": {
     "build": "npm run clean && npm run build:ts && npm run build:copy",
-    "build:ts": "tsc -p tsconfig.json && tsc -p tsconfig.json -m esnext --outDir dist/esm/ -d false -declarationMap false",
-    "build:copy": "copyfiles README.md LICENSE src/package.json dist --flat && copyfiles src/rxjs-operators/package.json dist --up 1",
+    "build:ts": "tsc -p tsconfig-cjs.json && tsc -p tsconfig-esm.json",
+    "build:copy": "copyfiles README.md LICENSE src/package.json dist --flat && copyfiles src/rxjs-operators/package.json dist/cjs --up 1 && cp package-cjs.json dist/cjs/package.json && cp package-esm.json dist/esm/package.json",
```

Comment:
I have not seen this approach in other packages. Have you seen other packages do it like that?

Did you see this?
https://www.the-guild.dev/blog/support-nodejs-esm

Reply:
Hey, I'm new to JS/TS packaging and I can't say I really know what other packages do. I followed https://www.sensedeep.com/blog/posts/2021/how-to-create-single-source-npm-module.html